spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
```

Comment:
Not all the tests are referencing these lazy-variables, but some of them require the DB objects created inside the let-blocks. In such cases you need the `!˙ but if you have variables that depend on each other, it's enough to use the `!` on the top-level.